README.md

```diff
@@ -1376,6 +1392,32 @@ bucket. This will automatically delete files in your bucket that are older than
 permits.
 [You can read more about configuring a retention policy in S3 here.](https://docs.aws.amazon.com/AmazonS3/latest/userguide/how-to-set-lifecycle-configuration-intro.html)
 
+## Document Download
+
+Form-flow library allows users to either:
+
+1. Download individual files
+2. Download a zip folder with all the files associated with a submission.
+
+In order to download from any endpoint, the http session must have an attribute of "id" that matches
+the submission's UUID that you would like to download. There are
+
```

Comment:
I'd change "any" to "these".  "http" should probably be "HTTP" since it's an abbreviation.  This just ends at "There are" ... should there be more?